### ISSUE TITLE
ReDeads can still be stunned by Sun’s Song without soul

### ISSUE THREAD

**Issue by ScArWlvrne** [gamestabled/OoT3D_Randomizer#723]
Not much more than the title. I played Sun’s Song for the chest in the grave, and the ReDead was stunned. Unsure if this is intended behavior, but figured I’d open an issue just in case.

**Comment by HylianFreddy** [comment]
Yes, this is known. I only patched things that can normally kill enemies, so very specific actions that can only stun are not changed.

Do you think it would be better to make soulless ReDeads immune to the Sun's Song?
